cmd/neofs-cli/internal/common/exit.go

```diff
@@ -26,6 +32,7 @@ func ExitOnErr(cmd *cobra.Command, errFmt string, err error) {
 		_ = iota
 		internal
 		aclDenied
+		awaitTimeout
```

Comment:
@roman-khimov, @cthulhu-rider, @AliceInHunterland dont we want to try to separate exit codes b/w API errors and internal? e.g. API errors are 1-63, internal are 64-127?

Reply:
local/remote? Can be done.

Reply:
i'd keep as it is: 1 for internals and all other for protocol

await timeout is not an internal failure to me. Yeah, no dedicated code, but this is a system behavior

Reply:
@carpawell now it's tricky to separate errors into two ranges: is internal code the only code in the internal range now? I think it's a good idea if we are going to distinguish further into error codes by creating codes for the most common errors. However for now we have only 3, and the range of 1 of them is questionable. So maybe with the next error code, it will be easier to implement?

Reply:
> local/remote?

yeap, smth like that

> await timeout is not an internal failure to me

well, to me it is some intermediate state. i would not say it is a protocol or system error, it really depends on implementation: timeout is not a node's answer and we do not describe it somewhere, it is just what a regular neofs client is able to do

> However for now we have only 3

i would say it is about being scared of exit code limitations and not about our willing not to do more codes

> So maybe with the next error code, it will be easier to implement?

it will be senseless then if we approve that PR: codes will have been mixed already and changing exit codes is not a thing i will appreciate

but once again: it is not even a suggestion but i would go that way if i have to implement more codes

Reply:
Timeout is decided locally, it's not something coming from remote server. At the same time a table is sufficient to differentiate two groups even if they're mixed. And now 1 can be returned for both cases and separating them can be tricky (every SDK error needs to be checked for).